fmriprep/workflows/epi.py

```diff
@@ -636,7 +638,9 @@ def _aslist(in_value):
         return [in_value]
 
     gen_ref = pe.Node(GenerateSamplingReference(), name='GenNewMNIReference')
-    gen_ref.inputs.fixed_image = op.join(get_mni_icbm152_nlin_asym_09c(), '1mm_T1.nii.gz')
+    template_str = {'MNI152NLin2009cAsym': 'mni_icbm152_nlin_asym_09c'}[template]
```

Comment:
Could we move this dictionary into one central place? it is being used a couple times